src/mocks/company/handlers.ts

```diff
@@ -0,0 +1,18 @@
+import { http, HttpResponse } from 'msw';
+
+import type { CompanyListResponse } from '@/mocks/company/schemas.ts';
+
+export const companyHandlers = [
+  http.get<never, never, CompanyListResponse[]>('*/api/company-list', () => {
+    const companies = [
+      { id: '1', name: '1start', description: 'first description' },
+      { id: '2', name: 'up2', description: 'second description' },
+      { id: '3', name: 'startup', description: 'third description' },
+      { id: '4', name: 'startup4', description: 'fourth description' },
+      { id: '5', name: 'startup5', description: 'fifth description' },
+      { id: '6', name: 'startup6', description: 'sixth description' },
+    ];
+
```

Comment:
resolver를 따로 빼시는 게 더 좋을 거 같습니다. 로직이 너무 한 군데에 모여 있는 느낌?
그리고 companies를 entity 폴더에서 관리하는 게 더 나을 거 같습니다.companies 데이터가 다른 api에서도 사용될 수 있어서요!

Reply:
```suggestion
// 위에서 타입 관리
type echoResolver = {
  pretotypeAddUser: HttpResponseResolver<
    never,
    PretotypeAddUserRequest,
    PretotypeAddUserResponse
  >;
};

// 아래에는 로직만
export const echoResolver: echoResolver = {
  pretotypeAddUser: async ({ request }) => {
    const { email, isSubscribed } = await request.json();
    const date = new Date();
    const formattedDate = `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()} ${date.getHours()}:${date.getMinutes()}:${date.getSeconds()}`;
    const context = {
      email,
      isSubscribed,
      createdAt: formattedDate,
    };

    return HttpResponse.json(context, { status: 200 });
  },
};

```

이렇게 작성하면 가운데에 좀 지저분하게 타입이랑 로직이랑 섞인 걸 더 보기좋게 관리할 수 있습니다.

Reply:
반영했습니다!